### PR TITLE
Fixed bucket search

### PR DESCRIPTION
**Pull request by weslyvo81**
This pull request is about fixing bucket search and date type in fyodor.

### CODE REVIEW

File: kogitoq/gadaq/src/main/java/life/genny/gadaq/search/FilterGroupService.java [hunk 1]
```diff
@@ -1149,4 +1161,16 @@ public void sendListQuickSearchDropdown(QEventMessage msg) {
             log.error(ex);
         }
     }
+
+    /**
+     * Return being bucket or not
+     * @param code Sbe code
+     * @return being bucket or not
+     */
+    public boolean isBucketSbe(String code) {
```
Comment (jasperRob) [inline kogitoq/gadaq/src/main/java/life/genny/gadaq/search/FilterGroupService.java:1170]: replace with:
```java
    public boolean isBucketSbe(String code) {
        return SearchEntity.SBE_PROCESS.equals(code);
    }
```

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/BaseEntityService.java [hunk 2]
```diff
@@ -312,4 +312,32 @@ public void setupBaseEntity(String baseEntityCode, String... attributeCodeValue)
 		beUtils.updateBaseEntity(be);
 	}
 
+	/**
+	 * Initialize the base entity and assign actual definition
+	 * @param definitionCode Base definition code
+	 * @param detailDefCode Detail definition code
+	 * @return
+	 */
+	public String init(String definitionCode,String detailDefCode) {
```
Comment (jasperRob) [inline kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/BaseEntityService.java]: Please elaborate on the purpose of this method

File: kogitoq/gadaq/src/main/java/life/genny/gadaq/search/FilterGroupService.java (hunk 1 above)
Comment (weslyvo81) [reply kogitoq/gadaq/src/main/java/life/genny/gadaq/search/FilterGroupService.java:1170]: Solved

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/BaseEntityService.java (hunk 2 above)
Comment (weslyvo81) [reply kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/BaseEntityService.java]: Add more comments in the method

**Review by jasperRob — APPROVED** [review]

**Pull request merged**